auto_py_to_exe/web/index.html

```diff
@@ -66,6 +66,10 @@
                         </label>
                         <select id="language-selection"></select>
                     </span>
+                    <span id="theme" onclick="toggleTheme()">
+                        <img src="img/moon.svg" class="dark-theme-button" style="display: inline;" />
+                        <img src="img/sun.svg" class="light-theme-button" style="display: none;"/>
+                    </span>
```

Comment:
This toggle is looking a bit out of place - it might be better to use a simple label saying "Theme: Light" and "Theme: Dark" which toggles when clicked?

Configuring the on-click handler would be better suited in JavaScript - see my comment below for where this would make sense.

Also this should have a pointer cursor.